developer/src/test/auto/keyboard-info/test-valid.keyboard_info

```diff
@@ -1,4 +1,5 @@
 {
+    "id": "test-valid",
```

Comment:
Should this use underscores instead of dashes?

Reply:
ID doesn't matter for this test.